services/scabbard/libscabbard/src/migrations/diesel/postgres/migrations/2022-07-08-133500-add-consensus-2pc-views/down.sql

```diff
@@ -0,0 +1,18 @@
+-- Copyright 2018-2022 Cargill Incorporated
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- -----------------------------------------------------------------------------
+
+DROP VIEW all_consensus_2pc_actions;
```

Comment:
Need to re-consider the name as all_ isn't a good namespace prefix. Probably we should have a suffix instead.

Reply:
fixed